python/arepl_result_stream.py

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 'p') (D415)](https://app.codacy.com/manual/almenon214/AREPL-backend/pullRequest?prid=5418764)